language/move-prover/bytecode-to-boogie/tests/translator_tests.rs

```diff
@@ -34,55 +34,11 @@ fn test_struct() {
     test(NO_VERIFY, &["test_mvir/test-struct.mvir"]);
 }
 
-#[test]
-fn test_access_path() {
-    test(
-        &["--native-stubs"],
-        &[
-            &std_mvir("vector"),
-            &std_mvir("u64_util"),
-            &std_mvir("address_util"),
-            &std_mvir("hash"),
-            &std_mvir("libra_coin"),
-            &std_mvir("libra_time"),
-            &std_mvir("libra_transaction_timeout"),
-            &std_mvir("libra_account"),
-            "test_mvir/test-access-path.mvir",
-        ],
-    );
-}
-
-#[test]
-fn test_lib() {
-    test(
-        // We have not yet created .prover.bpl stubs for native functions in library, so instruct
-        // translator to generate them.
-        &["--native-stubs", "-B=-noVerify"],
-        &[
-            &std_mvir("vector"),
-            &std_mvir("u64_util"),
-            &std_mvir("address_util"),
-            &std_mvir("hash"),
-            &std_mvir("signature"),
-            &std_mvir("gas_schedule"),
-            &std_mvir("validator_config"),
-            &std_mvir("libra_coin"),
-            &std_mvir("libra_time"),
-            &std_mvir("libra_transaction_timeout"),
-            &std_mvir("libra_account"),
-            // TODO(wrwg): this currently fails with boogie compilation errors
-            //   call to undeclared procedure: Vector_contains (etc)
-            // &std_mvir("libra_system"),
-            "test_mvir/test-lib.mvir",
```

Comment:
`test-lib.mvir` contains a module with no declarations. I believe this test was to check that linking against the IR stdlib works. This is no longer relevant, so I killed the test. If I misunderstood the purpose of  this, let me know and I can bring it back.